protobuf/event/event.proto

```
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package event;
```

Buf (1.55.1)

Files with package "event" must be within a directory "event" relative to root but were in directory "protobuf/event".

(PACKAGE_DIRECTORY_MATCH)